vip-rest-api.php

```diff
@@ -106,8 +106,8 @@ public function list_sites() {
 			// Inflate raw list of site IDs, if available
 			if ( is_array( $_sites ) ) {
 				// Switch to the blog to ensure certain domain filtering is respected
-				foreach ( $_sites as $_site ) {
-					switch_to_blog( $_site );
+				foreach ( $_sites as $blog_id ) {
```

Comment:
Due to the risk of stomping on the `$blog_id` global, I opted for `$_site`. I'm fine with `$_blog_id`, but we shouldn't use `$blog_id`.